requirements_test.txt

```diff
@@ -10,3 +10,4 @@ pytest-profiling~=1.7
 pytest-xdist~=2.5
 # Type packages for mypy
 types-pkg_resources==0.1.3
+tox>=3
```

Comment:
?

Reply:
We need that for tbump (upgrading the doc)

Reply:
We might want to install it only in tbump, as it's going to slow down every pipelines for no good reason though. But we also suggest using tox in the doc...

Reply:
Ah okay. Well, I think this is fine. Most people will install it anyway at some point.